web/src/app/shared/converters/loi-converter/loi-data-converter.spec.ts

```diff
@@ -21,7 +21,7 @@ import {
   LocationOfInterest,
   PointOfInterest,
 } from '../../models/loi.model';
-import { toGeometry } from '../geometry-converter';
+import { toGeometry, GEOJSON_GEOMETRY_TYPES } from '../geometry-converter';
```

Comment:
We don't use a strict GeoJson format in our db, so it might be more appropriate just call this map something like `GEOMETRY_TYPES`.

Reply:
The name `GEOJSON` is to indicate that these follow the GeoJson naming. Does that make sense to you?